src/backend/vulkan/src/lib.rs

```diff
@@ -1294,11 +1318,16 @@ impl adapter::PhysicalDevice<Backend> for PhysicalDevice {
     }
 }
 
+struct DeviceExtensionFunctions {
+    draw_indirect_count: Option<vk::KhrDrawIndirectCountFn>,
```

Comment:
why don't we use the slightly more convenient https://docs.rs/ash/0.31.0/ash/extensions/khr/struct.DrawIndirectCount.html ?

Reply:
Damn, missed that, yes definitely. Always have trouble finding things in ash.

Reply:
I see that you pushed some code after the review, but this point still stands